orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -48,10 +50,12 @@ class MonitorKatoTask implements RetryableTask {
 
   private final Clock clock
   private final Registry registry
+  @Autowired private KatoService kato
```

Comment:
could you add this to the constructor instead

Reply:
yeeeeeahh... I tried once and wound up reverting it. I'll try again.